tests/test.spec.ts

```diff
@@ -14,7 +14,17 @@ import {
   TestSetup,
   USER_NAME,
 } from "./test-setup";
-
+import treeKill from "tree-kill";
+
+const killAllProcesses = async (process: ChildProcess) => {
+    if (process?.pid) {
+        treeKill(process.pid, "SIGKILL", (error) => {
+            if (error) {
+                throw error;
+            }
+        });
+    }
+};
```

Comment:
Re-throwing the error here like that isn't good 'promisifycation'. 
```suggestion
import { promisify } from 'node:util';
const treeKillAsPromised = promisify(treeKill);
const killAllProcesses = async (process: ChildProcess) => {
    if (process?.pid) {
      await  treeKillAsPromised(process.pid, "SIGKILL");
    }
};
```

Reply:
Fixed